score_threshold: 0.7
nms_threshold: 0.3
target_color: 1